examples/mint-repl/mint-repl.rs

```diff
@@ -0,0 +1,1110 @@
+        //        println!("  {}. {}", i, to_be_hex::<PublicKeyShare>(&sks.public_keys().public_key_share(i))?);
+        println!(
+            "  {}. {}",
+            i,
+            encode(&sks.public_keys().public_key_share(i).to_bytes())
+        );
+    }
+
+    println!(
+        "\n -- PublicKeySet --\n{}\n",
+        to_be_hex(&sks.public_keys())?
+    );
+
+    println!(
+        "\nSigning Threshold: {}  ({} signers required)\n",
+        sks.threshold(),
+        sks.threshold() + 1
+    );
+
+    Ok(())
+}
+
+/// Displays mint information in human readable form
+fn print_mintinfo_human(mintinfo: &MintInfo) -> Result<()> {
+    println!();
+
+    println!("Number of Mint Nodes: {}\n", mintinfo.mintnodes.len());
+
+    println!("-- Mint Keys --\n");
+    println!("SecretKeySet (Poly): {}\n", to_be_hex(&mintinfo.poly)?);
+
+    println!(
+        "PublicKeySet: {}\n",
+        to_be_hex(&mintinfo.secret_key_set.public_keys())?
+    );
+
+    println!("\n   -- SecretKeyShares --");
+    for i in (0..mintinfo.secret_key_set.threshold() + 2).into_iter() {
+        println!(
+            "    {}. {}",
+            i,
+            encode(&sks_to_bytes(&mintinfo.secret_key_set.secret_key_share(i))?)
+        );
+    }
+
+    println!("\n   -- PublicKeyShares --");
+    for i in (0..mintinfo.secret_key_set.threshold() + 2).into_iter() {
+        // the 2nd line matches ian coleman's bls tool output.  but why not the first?
+        //        println!("  {}. {}", i, to_be_hex::<PublicKeyShare>(&sks.public_keys().public_key_share(i))?);
+        println!(
+            "    {}. {}",
+            i,
+            encode(
+                &mintinfo
+                    .secret_key_set
+                    .public_keys()
+                    .public_key_share(i)
+                    .to_bytes()
+            )
+        );
+    }
+
+    println!(
+        "\n   Required Signers: {}   (Threshold = {})",
+        mintinfo.secret_key_set.threshold() + 1,
+        mintinfo.secret_key_set.threshold()
+    );
+
+    println!("\n-- Genesis DBC --\n");
+    print_dbc_human(&mintinfo.genesis, true)?;
+
+    println!("\n");
+
+    println!("-- SpendBook --\n");
+    for (dbchash, _tx) in mintinfo.mintnode()?.spendbook.transactions.iter() {
+        println!("  {}", encode(&dbchash));
+    }
+
+    println!();
+
+    Ok(())
+}
+
+/// displays Dbc in human readable form
+fn print_dbc_human(dbc: &DbcOwned, outputs: bool) -> Result<()> {
+    println!("id: {}\n", encode(dbc.inner.name()));
+    println!("amount: {}\n", dbc.inner.content.amount);
+    println!("output_number: {}\n", dbc.inner.content.output_number);
+    println!("owner: {}\n", to_be_hex(&dbc.owner)?);
+
+    // dbc.content.parents and dbc.transaction.inputs seem to be the same
+    // so for now we are just displaying the latter.
+    // println!("parents:");
+    // for p in &dbc.content.parents {
+    //     println!("  {}", encode(p))
+    // }
```

Comment:
Yep, they are the same for valid DBC's. One of the validation steps is checking that transaction inputs matches content inputs.